src/shippingservice/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-FROM --platform=${BUILDPLATFORM} rust:1.73 as builder
```

Comment:
Is there a issue with 1.73? We should still support this, if not we need to check our MSRV setup

Reply:
No issue, I'm just updating to latest rust version.